### ISSUE TITLE
Missing Source Files from Database for Chromium

### ISSUE THREAD

**Issue by creative16** [github/codeql-cli-binaries#87]
This is a continuation of Issue #39.

When trying to build the database for Chromium (a more recent checkout) using 2.6.3, it seems that none of the source files were successfully extracted into the database.

A quick look into build-tracer.log revealed many instances of the following lines:

```
[E ...] Checking whether C compilation already happened.
[E ...] Checking for tag c-compilation-happened
[E ...] Checking CODEQL_TRACER_DB_ID ... (some string of hex numbers)
[E ...] Tag exists.
[E ...] Exiting as C compilation already happened.
```

Not sure if this gives a clue for the root cause of the issue, but any assistance/advice would be much appreciated. Thanks!

**Comment by creative16** [comment]
I was using checkout b35194229662a74ee2a497ad5964d90e6cb87de3, on Windows 10 Build 19041 x64. I used the following commands to create the database:

```
> gn gen out\Win --args="is_component_build = false is_debug = false enable_nacl = false symbol_level = 0 blink_symbol_level = 0"
> codeql database create C:\Database\chromium_db --language=cpp --command="ninja -C out\Win chrome -j 6"
```

I've also tried the above commands using 2.5.2 and 2.5.7. The problem manifested itself in 2.5.7, but not in 2.5.2, so the issue probably surfaced sometime after 2.5.2.

**Comment by sashabu** [comment]
Thanks for your patience. This is fixed in 2.7.5.
